subsystems/visionSensor.py

```diff
@@ -44,7 +42,7 @@ def createChannel(self):
         vision_ip = self.vision_ip
         vision_port = self.vision_port
         try:
-            self.channel = UDPChannel(local_ip="10.10.76.2", local_port=5880, 
```

Comment:
We also need to explicitly test to make sure this failure mode will leave a None in the self.channel.    Might need to explicitly set it to None if the UDPChannel creation throws an exception.  (and maybe it should).    Not certain otherwise how to recover from a failure to create something.